namespaces/init.go

```diff
@@ -82,6 +82,12 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 		}
 	}
 
+	cloneFlags := GetNamespaceFlags(container.Namespaces, true)
```

Comment:
I was thinking about this. mount.InitializeMountNamespace() can't be executed without a mount namespace even if container.MountConfig is nil.  I would like to have these checks near the code which they affect. But I don't insist on this and if mount.InitializeMountNamespace() isn't convinced you, I can rework this series. Thanks.